huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -68,12 +69,59 @@ var AdvisorAssignmentsView = React.createClass({
               {this.renderAssignmentRows()}
             </table>
           </div>
-          <div className="tablemenu footer" />
+          <div className="tablemenu footer">
+            <Button
+              color="green"
+              size="small"
+              loading={this.state.loading}
+              onClick={this._handleClick}>
+              Finalize Assignments
+            </Button>
+          </div>
         </form>
       </InnerView>
     );
   },
 
+  _handleClick: function(event) {
+    this.setState({loading: true});
+    var user = this.props.user.getData();
+    var school = this.props.user.getSchool();
+    var elements = document.getElementsByTagName("input");
+    var assignList = [];
+    for (var i=0; i<elements.length; i++) {
+      if (elements[i].checked) {
+        assignList.push(elements[i].getAttribute("name"));
+      }
+    }
+    window.alert("worked");
+    $.ajax({
+      type: 'POST',
+      url: '/api/schools/' + school.id + '/finalize/',
+      data: assignList,
```

Comment:
The data should JSON-encoded (If I'm not mistaken). Check out other requests and see if that's the case.